.github/workflows/build.yaml

```diff
@@ -137,58 +140,33 @@ jobs:
             type=raw,value=latest,enable={{is_default_branch}}
       - name: Build and push base
         if: ${{ inputs.baseTarget != '' }}
-        uses: docker/build-push-action@v3
+        uses: scientist-softserv/actions/build-and-push@make_actions_more_like_local
         with:
-          build-args: |
-            APP_PATH=${{ inputs.subdir }}
-          cache-from: type=registry,ref=${{ env.REGISTRY }}/${{ env.REPO_LOWER }}/base:latest
-          context: .
-          file: ${{ inputs.subdir}}/Dockerfile
-          platforms: ${{ inputs.platforms }}
-          push: true
-          tags: |
-            ${{ steps.meta-base.outputs.tags }}
-            ${{ env.REGISTRY }}/${{ env.REPO_LOWER }}/base:${{ env.TAG }}
-          target: ${{ inputs.baseTarget }}
+          type: base
+          location: /base
+          subdir: ${{ inputs.subdir }}
+          tags: ${{ steps.meta-base.outputs.tags }}
       - name: Build and push web
         if: ${{ inputs.webTarget != '' }}
-        uses: docker/build-push-action@v3
+        uses: scientist-softserv/actions/build-and-push@make_actions_more_like_local
         with:
-          build-args: |
-            APP_PATH=${{ inputs.subdir }}
-          cache-from: type=registry,ref=${{ env.REGISTRY }}/${{ env.REPO_LOWER }}:latest
-          context: .
-          file: ${{ inputs.subdir}}/Dockerfile
-          platforms: ${{ inputs.platforms }}
-          push: true
-          tags: |
-            ${{ steps.meta-web.outputs.tags }}
-            ${{ env.REGISTRY }}/${{ env.REPO_LOWER }}:${{ env.TAG }}
-          target: ${{ inputs.webTarget }}
+          type: web
+          location: ""
+          subdir: ${{ inputs.subdir }}
+          tags: ${{ steps.meta-base.outputs.tags }}
       - name: Build and push worker
         if: ${{ inputs.workerTarget != '' }}
-        uses: docker/build-push-action@v3
+        uses: scientist-softserv/actions/build-and-push@make_actions_more_like_local
         with:
-          build-args: |
-            APP_PATH=${{ inputs.subdir }}
-          cache-from: type=registry,ref=${{ env.REGISTRY }}/${{ env.REPO_LOWER }}/worker:latest
-          context: .
-          file: ${{ inputs.subdir}}/Dockerfile
-          platforms: ${{ inputs.platforms }}
-          push: true
-          tags: |
-            ${{ steps.meta-worker.outputs.tags }}
-            ${{ env.REGISTRY }}/${{ env.REPO_LOWER }}/worker:${{ env.TAG }}
-          target: ${{ inputs.workerTarget }}
+          type: worker
+          location: /worker
```

Comment:
Is type target? Noting I am confused here, but may work through it and come back.